test/powershell/Language/Scripting/Debugging/Debugging.Tests.ps1

```diff
@@ -324,7 +324,7 @@ Describe "It should be possible to reset runspace debugging" -tag "Feature" {
         $breakpoints.Count | Should -Be 2
     }
     It "The breakpoint Should have been hit" {
-        $completed | Should -BeTrue
+        $completed | Should -BeExactly $true
```

Comment:
`Should -BeTrue` asserts that the value evaluates to true, while `Should -BeExactly $true` asserts that the value *is* true (is of type Boolean). We want to test that `Wait-UntilTrue` actually returns true not some output that evaluates to true.

Reply:
I see your point. I think it is not full fix. We fix only false positives if a script block returns extra thing but we don't fix false negatives if a script block returns  extra thing like `$true`.
I believe a right way is to fix `Wait-UntilTrue` so that exclude extra output at all.

Reply:
I agree with @iSazonov here.

Reply:
Reverted.